src/ipLoadBalancing/farm/ipLoadBalancing-farm-server.lexi.service.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+_.forEach(["tcp", "udp", "http"], function (type) {
```

Comment:
You should use arrow function `() => {}`

Reply:
We've been told we can't use ES6 in ovh-api-services.

Reply:
I guess that we can use ES6 since this project has been opensourced and therefore it is already transpiled to ES5 when imported from other projects...

Reply:
The problem is that https://github.com/ovh-ux/ovh-api-services/blob/master/.eslintrc.json prevents us to do that for now ;)